test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -400,6 +440,80 @@ void ProtoConverter::visit(VarDecl const& _x)
 	}
 }
 
+void ProtoConverter::visit(MultiVarDecl const& _x)
+{
+	m_output << "let ";
+	vector<string> varNames;
+	// We support up to 4 variables in a single
```

Comment:
why 4?

Reply:
It's an arbitrarily chosen limit. `4` mostly because we don't want to run into stack too deep assertions.